sdk/typespec/typespec_client_core/Cargo.toml

```diff
@@ -43,6 +45,9 @@ tokio.workspace = true
 tracing.workspace = true
 tracing-subscriber.workspace = true
 typespec_macros.path = "../typespec_macros"
+wasm-bindgen-test = "0.3"
+getrandom = { version = "0.3" , features = ["wasm_js"]}
+uuid = { version = "1", features = ["v4", "js"] }
```

Comment:
All versions need to be declared in the workspace `Cargo.toml`. You can add features in crates like this if necessary e.g.,

```toml
uuid = { workspace = true, features = ["js", "v4"] }
```

That would union the feature set.